composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/MySpotsView.kt

```diff
@@ -57,134 +75,264 @@ fun MySpotsView(mySpotsViewModel: MySpotsViewModel, onAddSpotClick: () -> Unit)
 
         Box(modifier = Modifier.fillMaxSize()) {
 
-            Column(modifier = Modifier
-                .fillMaxSize()
-                .background(color = Color(0xFFF5F5F5))
-                .align(Alignment.Center)
-                .padding(16.dp)
+            Box(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .background(color = Color(0xFFF5F5F5))
+                    .align(Alignment.Center)
+//                    .padding(horizontal = 16.dp),
             ) {
 
                 if (spot == null) {
-                    Text(text = "No spots yet, add one!",
-                        modifier = Modifier.padding(16.dp).align(Alignment.CenterHorizontally),
-                        color = Color.Black,
-                        textAlign = TextAlign.Center,
-                    )
+                    Row(
+                        modifier = Modifier.fillMaxWidth()
+                    ) {
+                        Text(
+                            text = "No spots yet, add one!",
+                            modifier = Modifier.padding(16.dp),
+                            color = Color.Black,
+                            textAlign = TextAlign.Center,
+                        )
+                    }
                 } else {
 
-                    Box(
-                        modifier = Modifier.fillMaxWidth().padding(16.dp),
-                        contentAlignment = Alignment.Center
-                    ) {
-                        Row(modifier = Modifier.fillMaxWidth().align(Alignment.CenterStart)) {
-                            Image(
-                                modifier = Modifier.size(24.dp),
-                                imageVector = vectorResource(Res.drawable.accessibility_on),
-                                contentDescription = null
-                            )
-                            Spacer(modifier = Modifier.size(16.dp))
-                            Image(
-                                modifier = Modifier.size(24.dp),
-                                imageVector = vectorResource(Res.drawable.weather_on),
-                                contentDescription = null
-                            )
-                        }
+                    Column(modifier = Modifier.verticalScroll(rememberScrollState())) {
+                        Column(modifier = Modifier.padding(32.dp)) {
```

Comment:
Should maybe reduce this padding to 16.dp to make it more consistent with other pages

Reply:
This page had a double 16.dp before, that's why I was using 32 here. Just updated it (plus the other references)